### PR TITLE
06 html layout

### PR DESCRIPTION
**Pull request by BasharahBatool**
@nehero lesson 06 completed

### CODE REVIEW

**Comment by BasharahBatool** [comment]
@nehero lesson 06 completed

File: lessons/06-html-layout/posts/post-2.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<head>
+  <Title>News Article 2</Title>
```
Comment (ozziexsh) [inline lessons/06-html-layout/posts/post-2.html:3]: Watch out for capitals, everything should be lowercase, e.g. `<title>`

**Review by ozziexsh — COMMENTED** [review]
Looks good, just don't forget about the `<html>` boilerplate 

```
<html>
  <head>
  </head>
  <body>
  </body>
</html>
```

File: lessons/06-html-layout/posts/post-2.html (hunk 1 above)
Comment (BasharahBatool) [reply lessons/06-html-layout/posts/post-2.html:3]: Thanks for mentioning ....both corrections done

**Pull request merged**